src/sider/arbeidsuforhet/ArbeidsuforhetSide.tsx

```diff
@@ -30,6 +31,7 @@ export function ArbeidsuforhetSide({ children }: Props): ReactElement {
           </Tredelt.FirstColumn>
           <Tredelt.SecondColumn>
```

Comment:
Denne er sticky og må scrolles hvis innholdet er lenger enn 100vh. Med mange sykmeldinger i 'utdrag fra sykefraværet' kan 'rundskrivLenke' være vanskelig å finne for bruker.